kitsune/users/migrations/0034_delete_old_account_events.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+from django.utils import timezone
+from datetime import timedelta
+
+
+def delete_old_account_events(apps, schema_editor):
+    AccountEvent = apps.get_model('users', 'AccountEvent')
```

Comment:
In order to keep this code DRY and have a single place where we delete accounts,  you can use the `call_command` function to call the management command that you wrote

Reply:
Done